src/main/java/com/google/sps/servlets/AddAvailabilityServlet.java

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019 Google LLC
```

Comment:
I noticed that you implemented the doPost method of the AvailabilityServlet for **getting** a tutor's availability. Should that implementation be in the doGet method? If you can do that, I think it would make sense to move the  AddAvailabilityServlet's doPost method to the AvailabilityServlet (kind of like how we did it for the comments in the starter project).

Reply:
I implemented it as a post doPost method so that I could pass down the tutor's id. I could not figure out how to pass down the tutorID in a GET request.

Reply:
I left a comment below on how to do this. Let me know if it works!

Reply:
Thanks! I'll try using it